### PR TITLE
UPLOAD-1352/update report structs to reflect schema changes

### PR DESCRIPTION
**Pull request by cfarmer-fearless**
These are new changes following the schema defined here: https://github.com/CDCgov/data-exchange-messages/blob/foundation/reports/reports/README.md.

These will be the new baseline for the report schemas.  Any changes to the schema going forward will be reflected in a new version.

### CODE REVIEW

File: upload-server/pkg/reports/builder.go [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+			DataStreamRoute:   metadata.GetDataStreamRoute(b.Manifest),
+			Jurisdiction:      metadata.GetJurisdiction(b.Manifest),
+			DexIngestDatetime: metadata.GetDexIngestDatetime(b.Manifest),
+			ContentType:       "application/json",
+			DispositionType:   b.DispositionType,
+			StageInfo: ReportStageInfo{
+				Issues:           b.Issues,
+				Stage:            b.Stage,
+				Service:          "", // TODO get from version package
+				Version:          "", // TODO get from version package
```
Comment (whytheplatypus) [inline upload-server/pkg/reports/builder.go:183]: finish this up and add repo

**Review by whytheplatypus — CHANGES_REQUESTED** [review]
this looks great.

Because of the generic in Report, a `GetContentBuilder` or similar function could be make too, which would mean a writer wouldn't have to instantiate both the contentbuilder and the report builder... not sure if that's worth a change, just an idea.

File: upload-server/pkg/reports/builder.go (hunk 1 above)
Comment (cfarmer-fearless) [reply upload-server/pkg/reports/builder.go:183]: It's a dependency of this PR: https://github.com/CDCgov/data-exchange-upload/pull/419

**Comment by cfarmer-fearless** [comment]
@whytheplatypus  I went ahead and removed content builder all together.  After reading your comment and looking at the code again, I decided it was redundant.  Thanks!

**Review by whytheplatypus — APPROVED** [review]

**Pull request merged**